src/assets/config.json

```diff
@@ -11,5 +11,8 @@
         "bsc": "https://ethparser-stage.harvestfi.builders"
     },
     "multipleSources": true,
-    "defaultNetwork": "eth"
+    "defaultNetwork": "eth",
+    "theGraph": {
+        "graphQlServerUrl": "https://api.thegraph.com/subgraphs/name/graphprotocol/graph-network-mainnet"
```

Comment:
we can use different subgraphs - name should be adjustable too
for example our thegraph is https://api.thegraph.com/subgraphs/name/harvestfi/harvest-finance
uniswap is https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2